pkg/services/dashboards/database/database.go

```diff
@@ -990,3 +990,14 @@ func (d *DashboardStore) GetDashboardTags(ctx context.Context, query *models.Get
 		return err
 	})
 }
+
+func (d *DashboardStore) HasActivePublicDashboard(ctx context.Context, query *models.HasActivePublicDashboardQuery) error {
+	return d.sqlStore.WithDbSession(ctx, func(dbSession *sqlstore.DBSession) error {
+		sql := "SELECT COUNT(*) FROM dashboard_public WHERE dashboard_uid=? AND is_enabled=1"
```

Comment:
I think `is_enabled` should be compared to `true` instead of 1. Not all dbs store boolean as ints.

Reply:
good call